nifi-ngsi-bundle/nifi-ngsi-processors/src/main/java/org/apache/nifi/processors/ngsi/ngsi/backends/PostgreSQLBackend.java

```diff
@@ -96,18 +96,18 @@ else if (subAttribute.getAttrValue() instanceof Number){
 
     private String encodeAttributeToColumnName(String attributeName, String datasetId, String datasetIdPrefixToTruncate) {
         String encodedName = NGSIEncoders.encodePostgreSQL(attributeName) + (!datasetId.equals("") ? "_" + NGSIEncoders.encodePostgreSQL(datasetId.replaceFirst(datasetIdPrefixToTruncate, "")) : "");
-        return NGSIEncoders.truncateToMaxSize(encodedName);
+        return NGSIEncoders.truncateToMaxSize(encodedName).toLowerCase();
```

Comment:
are these 3 `toLowerCase()` related to the current feature?

Reply:
yes beacause when we get columnName in DB the name are in lowercase so for all to be identical it is necessary to put lowercase

Reply:
ok